src/util/ClientServiceUtility.java

```diff
@@ -0,0 +1,44 @@
+package util;
+
+import entities.BaseEntity;
+import entities.User;
+
+public final class ClientServiceUtility {
+  public static BaseEntity updateBaseEntity(BaseEntity baseEntityToUpdate, BaseEntity updated) {
+    baseEntityToUpdate.setUpdatedAt(
+        updated.getUpdatedAt() != null
+            ? baseEntityToUpdate.getUpdatedAt()
```

Comment:
Can you use optionals for all these properties?

Reply:
solved